checkov/circleci_pipelines/checks/image_version_not_hash.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+from typing import Any
+
+from checkov.circleci_pipelines.base_circleci_pipelines_check import BaseCircleCIPipelinesCheck
+from checkov.common.models.enums import CheckResult
+from checkov.yaml_doc.enums import BlockType
+
+
+class ImageReferenceHashVersion(BaseCircleCIPipelinesCheck):
+    def __init__(self) -> None:
+        name = "Ensure the pipeline image version is referenced via hash not arbitrary tag."
+        id = "CKV_CIRCLECIPIPELINES_2"
+        super().__init__(
+            name=name,
+            id=id,
+            block_type=BlockType.ARRAY,
+            supported_entities=['jobs.*.docker[].{image: image, __startline__: __startline__, __endline__:__endline__}']
+        )
+
+    def scan_conf(self, conf: dict[str, Any]) -> tuple[CheckResult, dict[str, Any]]:
+        if not isinstance(conf, dict):
+            return CheckResult.PASSED, conf
+        image = conf.get("image", None)
+        if not image:
+            return CheckResult.PASSED, conf
+        if isinstance(image, str):
+            if "@" in image:
+                return CheckResult.PASSED, conf
+            if "latest" in image:
+                return CheckResult.PASSED, conf
```

Comment:
```suggestion
                return CheckResult.UNKNOWN, conf
```
fits here better, when you actually want to ignore the `latest` tag case.

Reply:
Ahh, tried SKIP with issues, unknown is better! Thanks!

Reply:
This will change passed # so i'll have to tweak runner tests too